doc/en/usage.rst

```diff
@@ -208,22 +183,9 @@ More than one character can be used, so for example to only see failed and skipp
     =========================== test session starts ============================
     platform linux -- Python 3.x.y, pytest-4.x.y, py-1.x.y, pluggy-0.x.y
     rootdir: $REGENDOC_TMPDIR, inifile:
-    collected 2 items
-
-    test_examples.py Fs                                                      [100%]
-
-    =================================== FAILURES ===================================
-    __________________________________ test_fail ___________________________________
+    collected 0 items
 
-        def test_fail():
-    >       assert 0
-    E       assert 0
-
-    test_examples.py:14: AssertionError
-    =========================== short test summary info ============================
-    FAIL test_examples.py::test_fail
-    SKIP [1] test_examples.py:21: Example
-    ===================== 1 failed, 1 skipped in 0.09 seconds ======================
```

Comment:
""

Reply:
Increased regendoc column size to 86, let's see

Reply:
Oh I completely misunderstood you, I thought you said the lines were wrongly wrapped, and it indeed looks like when you are in the "Conversation" tab in the PR. 😆 

Hmm now I have to investigate what happened here...

Reply:
iirc a workaround for now is to just put `#$` instead of `$` at the top and regendoc will ignore it and not regenerate it -- we can follow up later to fix those up